docs/Growing.md

```diff
@@ -415,12 +415,119 @@ with `ctx.redirectTo()`, you can use the flash, accessible through `ctx.flash`.
 ```js
 const flash = await ctx.flash();
 flash.message = 'Everything is fine.';
-return ctx.redirectTo('goodbye');
+await ctx.redirectTo('goodbye');
 ```
 
 Just remember that all session data gets serialized to JSON and stored in encrypted cookies, which usually have a `4096`
 byte (4KiB) limit, depending on browser.
 
+## Final prototype
+
+A final `myapp.js` prototype passing all of the tests above could look like this.
+
+```js
+import mojo from '@mojojs/core';
+import Users from './models/users.js';
+
+// Set cookie secret
+export const app = mojo({secrets: ['Mojolicious rocks']});
+
+app.models.users = new Users();
+
+// Main login action
+app.any('/', async ctx => {
+  // Query parameters
```

Comment:
`Query or POST parameters`.

Reply:
Fixed here and in previous use in the Growing guide.